server/controllers/expenses.js

```diff
@@ -35,7 +36,17 @@ export const create = (req, res, next) => {
 /**
  * Get an expense.
  */
-export const getOne = (req, res) => res.json(req.expense.info);
+export const getOne = (req, res) => {
+  auth.canEditExpense(req, res, (e, canEditExpense) => {
```

Comment:
Thinking more about it, it's hard to add that info to encompass all expenses. The right place is a lib function that both auth and controllers can use. `canEditExpense(user, expense)`.